src/i18n/pt-br.js

```diff
@@ -0,0 +1,445 @@
+        spendKey: "Chave de gasto",
+        startingDaemon: "Começando serviço",
+        startingWallet: "Começando carteira",
+        switchToDateSelect: "Mudar para data de seleção",
+        switchToHeightSelect: "Mudar para altura de seleção",
+        transactionID: "ID da Transação",
+        transactions: {
+            amount: "Quantidade",
+            description: "{type} transação",
+            fee: "Taxa",
+            paidBySender: "pago por remetente",
+            received: "Recebido",
+            sent: "Enviado",
+            sentTo: "{type} transacão enviada a",
+            timestamp: "Data e hora",
+            types: {
+                all: "Todos",
+                incoming: "Entradas",
+                outgoing: "Saídas",
+                pending: "Pendentes",
+                pendingIncoming: "Entradas pendentes",
+                pendingOutgoing: "Saídas pendesntes",
+                miner: "Mineiro",
+                serviceNode: "Nódulo de Serviço",
+                governance: "Governança",
+                stake: "Participação",
+                failed: "Falhou"
+            }
+        },
+        unspentOutputs: "Outputs não-gastos",
+        userNotUsedAddress: "Não utilizou este endereço",
+        userUsedAddress: "Já utilizou este endereço",
+        viewKey: "Chave de visualização",
+        viewOnlyMode: "Modo de visualização apenas. Por favor carregue a carteira completa para poder enviar moedas."
+    },
+    titles: {
+        addressBook: "Livro de endereços",
+        addressDetails: "Detalhes do endereço",
+        changePassword: "Mudar palavra-passe",
+        configure: "Configure",
+        privateKeys: "Chaves privadas",
+        rescanWallet: "Re-examinar carteira",
+        serviceNode: {
+            registration: "REGISTO",
+            staking: "PARTICIPAÇÃO",
+            unlock: "DESBLOQUEAR"
+        },
+        settings: {
+            title: "Configurações",
+            tabs: {
+                general: "Geral",
+                language: "Língua",
+                peers: "Pares"
+            }
+        },
+        transactionDetails: "Detalhes das transações",
+        transactions: "Transações",
+        wallet: {
+            createNew: "Criar nova carteira",
+            createdOrRestored: "Carteira criada/restaurada",
+            importFromFile: "Importar carteira a partir de ficheiro",
+            importFromLegacyGUI: "Importar carteira a partir da antiga carteira GUI (legado)",
+            importFromOldGUI: "Importar carteira a partir da antiga carteira GUI",
+            restoreFromSeed: "Restaurar carteira a partir de semente",
+            restoreViewOnly: "Restaurar carteira visualização-apenas"
+        },
+        welcome: "Bem-vindo",
+        yourWallets: "Suas carteiras"
+    }
+}
```

Comment:
So just a general comment for two main fixes.

- replace "imágem" by "imagem" and "imágens" by "imagens" - that's an error

- replace "EXAMINAR" and "RE-EXAMINAR" by "ESCANEAR" and "REESCANEAR" since it is more correct imo

Some initial comments are on this suggestion.

Reply:
My addy:
`LWDYHoU8gptMQRXN6ybpGUffcydy7t92STvwKtw6nq8eVRtXezyE1JmRRMj7DV787k5ssZswR9B2PVe955VMrEiAABw53rX`

Reply:
As mentioned on other comments, all "imágens" has been fixed to adhere to the New Portuguese Orthographic Agreement (Acordo Ortográfico)

However, I do no agree with "ESCANEAR". It is a foreign terms that was absorbed and it pertains to digitising something, not to search or view. Also, it could clash with EXAMINAR in terms of consistency. 

As reference:
https://www.linguee.pt/ingles-portugues/traducao/scanning.html
As shown in this industry standard translation tool, the "antivirus is analysing" not "escaneando"

Of course, it can be used in IT slang due to similarities in phonetics, but the etymology of the word in Portuguese makes it semantically wrong to apply formally.

Reply:
Ok i have discussed with Kee about what they actually mean by scan here and it apparently is used for when you are syncing and when you are looking for a certain thing in the network.

So i don't think either of our words fits correctly.

This is tough cause in one context we should use the word "sincronizando" and in the other something like the verb "pesquisar", "buscar" or "procurar".

Reply:
Agreed. How about "Analisando" (analysing)? - It fits in between as it's slightly ambiguous without context, but in context it borrows significance. For example;

- Context A - sync) "Perderá o progresso da **sincronização** e terá de **reanalisar** o blockchain."

- Context B - search) "Para **encontrar** informações sobre a sua tx no blockchain, tem de **analisa-la** no block explorer"

Reply:
@KeeJef - was waiting for Luigy's opinion. I need to go over the the file once over to make changes. I will use **sincronização** where directly applicable, **analizar** when in context.

Reply:
> Ok i have discussed with Kee about what they actually mean by scan here, and it apparently is used for when you are syncing and when you are looking for a certain thing in the network.
> 
> So i don't think either of our words fits correctly.
> 
> This is tough cause in one context we should use the word "sincronizando" and in the other something like the verb "pesquisar", "buscar" or "procurar".

'Analisando' means that something (data) is already available for research and waited for be choosed.